vent/api/actions.py

```diff
@@ -388,12 +392,29 @@ def restore(self, backup_file):
             backup = Template(backup_file)
             options = ['repo', 'branch', 'version', 'built', 'namespace',
                        'path', 'groups', 'type', 'name', 'link_name',
-                       'pull_name']
+                       'pull_name', 'files']
             template_options = ['service', 'settings', 'docker', 'info',
                                 'gpu']
             backedup_tools = backup.constrained_sections({}, options +
                                                          template_options)
             for tool in backedup_tools:
+                # main means we're looking at vent.cfg
```

Comment:
there's more than just the `main` section now, so you'll probably want to keep the backup as two separate files.  probably making a directory for the backup that contains any and all files that need to be backed up.  currently backup keeps the config, but in the future it might need to backup images and containers and volumes, etc. so it will likely need to be more than just a single file anyway.